### ISSUE TITLE
locked SQLITE Database

### ISSUE THREAD

**Issue by RonnyB1978** [linuxserver/docker-duplicati#71]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I have installed Ubuntu Server 22.04 LTS and run Docker Compose there. The volumes are mapped to my Synology DS918. When I create the stack using Portainer, I get the error that the database is locked.

### Expected Behavior

correct mapping of all files

Thanks for help

### Steps To Reproduce

1. Docker install
2. Portainer install
3. Stack install
4. Error SQLITE

### Environment

```markdown
- OS:linux x86_64 Ubuntu 22.04.3 LTS
- Kernel 5.15.0-91-generic
- How docker service was installed: sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
- Portainer: docker run -d -p 8000:8000 -p 9443:9443 --name portainer --restart=always -v /var/run/docker.sock:/var/run/docker.sock -v portainer_data:/data portainer/portainer-ce:latest
```

### CPU architecture

x86-64

### Docker creation

```bash
---
version: "2.1"
services:
  duplicati:
    image: lscr.io/linuxserver/duplicati:latest
    container_name: duplicati
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/BERLIN
      - CLI_ARGS= #optional
    volumes:
      - var/lib/docker/volumes/duplicati/config:/config
      - var/lib/docker/volumes/duplicati/backups:/backups
      - var/lib/docker/volumes/duplicati/source:/source
    ports:
      - 8200:8200
    restart: unless-stopped
```

### Container logs

```bash
[migrations] started
[migrations] no migrations found
───────────────────────────────────────
      ██╗     ███████╗██╗ ██████╗ 
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝ 
   Brought to you by linuxserver.io
───────────────────────────────────────
To support LSIO projects visit:
https://www.linuxserver.io/donate/
───────────────────────────────────────
GID/UID
───────────────────────────────────────
User UID:    1000
User GID:    1000
───────────────────────────────────────
[custom-init] No custom files found, skipping...
A serious error occurred in Duplicati: System.Exception: Failed to create, open or upgrade the database.
Error message: The database file is locked
database is locked
  at Duplicati.Server.Program.GetDatabaseConnection (System.Collections.Generic.Dictionary`2[TKey,TValue] commandlineOptions) [0x00229] in <30a34d71126b48248d040dda634ddad9>:0 
  at Duplicati.Server.Program.RealMain (System.String[] _args) [0x001a7] in <30a34d71126b48248d040dda634ddad9>:0
```

**Comment by aptalca** [comment]
You should not use `/var/lib/docker` to store your data. That path is for the docker service to use

**Comment by j0nnymoe** [comment]
Also, if what you're saying is correct, using remote mounts for databases isn't going to work.

**Comment by RonnyB1978** [comment]
> mounts for databases isn't going to work

Hello j0nnymoe,

thanks for your information. I changed the bind for the config folder to local machine -> home/xxxx/duplicati/db -> and it works. ;-)

@aptalca : Thanks also to you. I mounted the folder /var/lib/docker/volumes to my NAS. So I hoped all data will be saved persistent. Isn't it? The only what doesn't work, was the SQLITE database. I have seen that it was stored on my NAS, but with 0kb. After changing above, it works great.

**Comment by RonnyB1978** [comment]
> mounts for databases isn't going to work

Hello j0nnymoe,

thanks for your information. I changed the bind for the config folder to local machine -> home/xxxx/duplicati/db -> and it works. ;-)

@aptalca : Thanks also to you. I mounted the folder /var/lib/docker/volumes to my NAS. So I hoped all data will be saved persistent. Isn't it? The only what doesn't work, was the SQLITE database. I have seen that it was stored on my NAS, but with 0kb. After changing above, it works great.
